gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemBase.java

```diff
@@ -276,6 +267,9 @@ public int getByteLength() {
   /** The fixed reported permission of all files. */
   private FsPermission reportedPermissions;
 
+
+  /** Instrumentation to get the statistics. */
+  private static GHFSInstrumentation instrumentation;
```

Comment:
Make it final and non-static, if it should not be shared between GHFSBase instances

Reply:
Updated